mlir/include/mlir/Dialect/Vector/Utils/VectorUtils.h

```diff
@@ -194,7 +194,7 @@ bool isLinearizableVector(VectorType type);
 /// for each dimension of the passed in tensor.
 Value createReadOrMaskedRead(OpBuilder &builder, Location loc, Value source,
                              ArrayRef<int64_t> readShape, Value padValue,
-                             bool useInBoundsInsteadOfMasking = true);
+                             bool useInBoundsInsteadOfMasking = false);
```

Comment:
Can we just drop the default value? From the name of the method, it is not clear about which path is default. So I think it is caller's responsibility to set the value explicitly. IIRC, we did not catch the behavior is just because all the uses set the value explicitly. So I'd suggest to just drop the default value.

Reply:
done, PTAL.